content/made-with-kicad/hades-fcs.adoc

```diff
@@ -0,0 +1,8 @@
++++
+title = "HADES Flight Control System"
+projectimage = "/img/madewithkicad/hades-fcs.jpg"
+projectdeveloper = "Philip Salmony"
+projecturl = "https://github.com/pms67/HadesFCS/"
++++
+
+link:https://github.com/pms67/HadesFCS[HADES FCS] is an open-source flight control system for unmanned aerial vehicles (UAVs) designed completely from scratch. This includes the hardware (4-layer PCBs designed with KiCAD), STM32 firmware (using FreeRTOS), Kalman filtering for state estimation, control algorithms, base station, communication protocol, flight simulator, signal processing, and much, much more.
```

Comment:
Maybe just strp `s designed with KiCAD` out as this is intended to go on a website where the boards are only made with kicad.

Reply:
Okay, cool thank you for taking the time to look through this!
I've changed the description and removed the 'designed with KiCad' part. I hope everything should be okay now!